docs/learn-transactions.md

```diff
@@ -0,0 +1,62 @@
+
+The collators of a parachain can but do not have to pay transaction costs, and those costs - if any - are internal to the chain. The economics of a parachain are up to the parachain to define, and whether or not the collators are incentivized to send proofs to the RC validators and process the parachain transactions is entirely up to the designers. For example, a parachain can but does not have to use the weight system.
+
+## Parathread Transactions
+
+[Parathreads](learn-parathreads) are exactly the same in terms of logic as parachains and as such the chain economics are once again up to the parathread designers. A key difference is that a parathread has to pay a fee for its block to be included in the Relay Chain and, thus, to progress. 
+
+Additionally, a parachain slot dedicated to hosting parathreads (a parathread pool) has a current theoretical maximum of 50 parathread blocks per slot being included in the Relay Chain submission, and if there are more than 50 actively writing parathreads in that parathread pool, they will enter an instant on-chain auction. This auction will determine which parathreads get to write their state to the RC in this particular slot and thus progress in their block production. The fee incurred by this auction is non-refundable, and is in fact similar to the fee system in Ethereum or Bitcoin. Indeed, we expect a fee market to arise in times of high demand.
+
+The fee is paid by the collators doing the submission, and the collators will have to gauge when it's worth it for them to try and win that per-block auction. It is because of this difference that we expect parathread collators to be incentivized by some on-parathread local economy. This also means the collators will have to be aware of a conversion rate between the token they're rewarded in and DOT/KSM in which the inclusion fee is paid.
+
+The economics of parathread inclusion are still being fleshed out, so the 50 parathread blocks per slot number may change along with the specific inclusion fee.
+
+Transactions occurring on the parathread itself are, as with parachains, specific to that environment and are not metered by the Relay Chain in any way. Collators can but do not have to be incentivized to process the transactions, and the chain itself can but does not have to have any metering built-in.
+
+## Smart Contract Transactions
+
+Some chains (threads or full parachains) will include a smart contract pallet. This would let third parties develop applications on that chain without having to have native code commit access to that chain's runtime. Transactions within that smart contract pallet might need to be metered somehow as well.
```

Comment:
```suggestion
Some chains (threads or full parachains) will include a smart contract pallet. This would let third parties develop applications on that chain without having to have native code commit access to that chain's runtime. Transactions within that smart contract pallet will need to be metered or otherwise access-controlled to prevent spam and block fullness.
```